miio/integrations/yeelight/__init__.py

```diff
@@ -265,13 +265,17 @@ def __init__(
         debug: int = 0,
         lazy_discover: bool = True,
         model: str = None,
+        light_type: YeelightSubLightType = YeelightSubLightType.Main,  # for now only Main support this is for future PR.
     ) -> None:
         super().__init__(ip, token, start_id, debug, lazy_discover, model=model)
         if Yeelight._spec_helper is None:
             Yeelight._spec_helper = YeelightSpecHelper()
             Yeelight._supported_models = Yeelight._spec_helper.supported_models
 
         self._model_info = Yeelight._spec_helper.get_model_info(self.model)
+        self._light_type = light_type
```

Comment:
```suggestion
        self._light_type = YeelightSubLightType.Main
```

Reply:
Ok, I am deleting this. But there are questions about this, can I do this in the future PR? 

As we said in the previous PR (https://github.com/rytilahti/python-miio/pull/1166#discussion_r741493032)
> * The main `Yeelight` instance controls the main light.
> * The sub lights (be it night light, or background one) use the same API as the main one, and they are exposed through the API in an easy-to-use manner. Potential downstreams like homeassistant should be able to have a single entity implementation, and the only difference between the main light and the sub lights should be based on the available features.

As I understand we should have the same API class for main and background lighting. And I don't understand how to do this without extending the constructor. Could you give me some advise (Then I`ll go correct way in the future PR)?